### PR TITLE
Release note Azure CLI 2.0.33

### PR DESCRIPTION
**Pull request by troydai**
@sptramer

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/troydai/azure-docs-cli-python/blob/patch-2/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-839) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-839) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C24%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201805241823048563-839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Oz9ANcMgu57asC1knk2lGLZ44P35UWD6w3WCerSDcjI%3D&st=2018-05-24T18%3A21%3A17Z&se=2018-06-24T18%3A26%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/troydai/azure-docs-cli-python/blob/patch-2/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-839) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-839) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C24%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201805241833014498-839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QqoNlEDiLVBadsrZXT6ZhejDe28nWpFQw9nXC2XgTYQ%3D&st=2018-05-24T18%3A31%3A20Z&se=2018-06-24T18%3A36%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -13,6 +13,76 @@ ms.devlang: azure-cli
...
+        * "requestedServiceObjectiveName" property is now readonly. To update service objective, use --service-objective parameter or set sku.name property.
+        * "edition" property is now readonly. To update edition, use --edition parameter or set sku.tier property.
+        * "elasticPoolName" property is now readonly. To update elastic pool, use --elastic-pool parameter or set elasticPoolId property.
+        * "maxSizeBytes" property is now an integer value instead of a string.
+    * Elastic pool respose objects:
+      * "edition", "dtu", "databaseDtuMin", and "databaseDtuMax" properties are now readonly. To update, use --edition, --capacity, --db-max-capacity, and --db-min-capacity parameters respectively.
+* Database, data warehouse, and elastic pool create and update commands now accept parameters to set the family (i.e. compute generation) and capacity (i.e. scale) aspects of performance level. Capacity can be used to set the scale of DTU-based editions (e.g. Basic, Standard, Premium), and family & capacity can be used to set the scale of vcore-based editions (e.g. GeneralPurpose and BusinessCritical).
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Require the name of these arguments.

Comment (twitchax) [reply docs-ref-conceptual/release-notes-azure-cli.md]: @troydai, can you look at this?

@sptramer, who should update the version number on the landing page?

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/troydai/azure-docs-cli-python/blob/patch-2/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-839) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-839) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C25%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201805251627333158-839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nyeiIO3wpkWwBE0YgbjAdGVPlz018cR%2FakJwZUHGSts%3D&st=2018-05-25T16%3A25%3A50Z&se=2018-06-25T16%3A30%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/release-notes-azure-cli.md (hunk 1 above)
Comment (sptramer) [reply docs-ref-conceptual/release-notes-azure-cli.md]: I update the version numbers on the installation page.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/install-azure-cli.md](https://github.com/troydai/azure-docs-cli-python/blob/patch-2/docs-ref-conceptual/install-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest&branch=pr-en-us-839) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-839) (azure-cli-2017-03-09-profile) |
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/troydai/azure-docs-cli-python/blob/patch-2/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-839) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-839) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C25%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201805251702148090-839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=W%2F7ISkEl65li3lYlDkmq2KDah7ubUcsVRawSbsW4G1s%3D&st=2018-05-25T17%3A00%3A25Z&se=2018-06-25T17%3A05%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**